### DISCUSSION TITLE
"booted: You are not using the same version as this server"

### DISCUSSION THREAD

**Discussion by yeahnotmuch** [Pryaxis/TShock#1827] [General]
I have had a server for a few days, and all of a sudden it won't let me connect. It keeps saying, "booted: You are not using the same version as this server," I believe I have updated both Terraria, and Tshock to the latest versions, since steam doesn't give me any option to update it more, and I have Pre4 of Tshock. If this is an easy to fix bug, my apologies but I am new to servers and have little experience.

**Comment by ColinBohn** [comment]
Terraria was just updated on Steam to version 1.4.0.3, and the TShock devs are currently working on patching up to this version. A new release will be created as soon as it is ready, so keep an eye on the releases page for a new build: https://github.com/Pryaxis/TShock/releases
